### PR TITLE
Update/wallaby instances

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes https://github.com/filecoin-project/filecoin-docs/issues/1571

### CODE REVIEW

**Comment by johnnymatthews** [comment]
This'll get merged, pending no errors from Fleek.

**Comment by johnnymatthews** [comment]
Had to revert back to `v0` to APIs. This is ready for review.

File: content/en/developers/infrastructure/networks/index.md [hunk 1]
```diff
@@ -22,29 +21,187 @@ aliases:
...
+- [Wallaby](#wallaby), an early smart-contract and actor test network for bleeding edge [Filecoin Virtual Machine]({{< relref "filecoin-virtual-machine" >}}) deployments.
+
+{{< alert  >}}
+Check the status of each network and subscribe to updates at [status.filecoin.io](https://status.filecoin.io).
+{{< /alert >}}
 
 ### Mainnet
 
-Mainnet is the live production network that all nodes on the Filecoin network are connected to.
+Mainnet is the primary Filecoin network. Mainnet began on block 148,888. It supports 32 GiB and 64 GiB sectors.
```
Comment (jennijuju) [inline content/en/developers/infrastructure/networks/index.md:37]:
```suggestion
Mainnet is the primary Filecoin network. Mainnet began on block 148,888. It supports 32 GiB and 64 GiB sectors.
```

Is there a reason why we added the liftoff epoch?

File: content/en/developers/smart-contracts/how-tos/add-to-metamask/index.md [hunk 2]
```diff
@@ -62,55 +44,30 @@ Before we get started, you'll need the following:
...
+1. Click **Add a network**.
+1. Scroll down and click **Add a network manually**.
 1. Enter the following information into the fields:
 
     | Field | Value |
     | --- | --- |
-    | Network name | `Filecoin Wallaby` |
-    | New RPC URL | `https://wallaby.node.glif.io/rpc/v0` |
+    | Network name | `Filecoin Hyperspace` |
+    | New RPC URL | `https://api.hyperspace.node.glif.io/rpc/v0` |
```
Comment (jennijuju) [inline content/en/developers/smart-contracts/how-tos/add-to-metamask/index.md:58]: have talked to glif - confirmed will only serve eth api on v1 after todays patch, so lets update the endpoint

File: content/en/developers/smart-contracts/how-tos/add-to-metamask/index.md [hunk 3]
```diff
@@ -62,55 +44,30 @@ Before we get started, you'll need the following:
...
-
-1. Review the values in the fields and click **Save**:
-
-    ![](entering-network-options.png)
-
-1. The Wallaby testnet should now be shown in your MetaMask window:
-
-    ![](wallaby-network-added.png)
+    - Glif Explorer: `https://explorer.glif.io/?network=hyperspace`
+    - Filscan: `https://explorer.glif.io/?network=hyperspace`
```
Comment (jennijuju) [inline content/en/developers/smart-contracts/how-tos/add-to-metamask/index.md:65]: this points glif?

**Review by jennijuju — CHANGES_REQUESTED** [review]

File: content/en/developers/reference/json-rpc/eth/index.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ This method is intended to return a list of addresses owned by client. However,
 Example:
 
 ```curl
-curl --location --request POST 'https://wallaby.node.glif.io/rpc/v0' \
+curl --location --request POST 'https://api.hyperspace.node.glif.io/rpc/v0' \
```
Comment (jennijuju) [inline content/en/developers/reference/json-rpc/eth/index.md:26]: I am not a huge fan  that the docs should point in testnet by defualt - do we plan to change this back when fevm launch in mainnet? if so, can we open an issue to make sure we dont forget about it?

File: content/en/developers/smart-contracts/how-tos/add-to-metamask/index.md [hunk 5]
```diff
@@ -62,55 +44,30 @@ Before we get started, you'll need the following:
...
+1. Scroll down and click **Add a network manually**.
 1. Enter the following information into the fields:
 
     | Field | Value |
     | --- | --- |
-    | Network name | `Filecoin Wallaby` |
-    | New RPC URL | `https://wallaby.node.glif.io/rpc/v0` |
+    | Network name | `Filecoin Hyperspace` |
+    | New RPC URL | `https://api.hyperspace.node.glif.io/rpc/v0` |
     | Chain ID | `31415` |
```
Comment (jennijuju) [inline content/en/developers/smart-contracts/how-tos/add-to-metamask/index.md:59]:
```suggestion
    | Chain ID | `3141` |
```

**Comment by johnnymatthews** [comment]
Commits changes and suggestions from @jennijuju. Can merge, pending a 2nd approval.

File: content/en/networks/overview/available-networks/index.md [hunk 6]
```diff
@@ -200,3 +200,10 @@ Here is a list of networks that are now deprecated:
 | Name | Description | Deprecated on |
 | --- | --- | --- |
 | Nerpa | A test network with tiny sector sizes. This network was for app developers to test the very basic functionality of their applications before moving over to the calibration testnet. | 2021-08-16 |
+
+## Block explorers
```
Comment (jennijuju) [inline content/en/networks/overview/available-networks/index.md:204]:
```suggestion
### Block explorers
```

File: content/en/developers/infrastructure/networks/index.md [hunk 7]
```diff
@@ -22,26 +21,184 @@ aliases:
...
+**Bootstrap peers**:
+
+```plaintext
+/dns4/de0.bootstrap.wallaby.network/tcp/1337/p2p/12D3KooWHAvUVk5XuxSwi2dNLWbTDDRSGeHxMuWdQ3SQpRuNHbLz
+/dns4/de1.bootstrap.wallaby.network/tcp/1337/p2p/12D3KooWBRqtxhJCtiLmCwKgAQozJtdGinEDdJGoS5oHw7vCjMGc
+/dns4/ca0.bootstrap.wallaby.network/tcp/1337/p2p/12D3KooWCApBpUk7EX9pmEfyky1gKC6N2KJ74S1AwFfvnkDqw3pK
+/dns4/sg0.bootstrap.wallaby.network/tcp/1337/p2p/12D3KooWLnYqr4hRoNHBJQVXsFGkDoKuoVfw5R2ASw1bHzrWU5Px
+```
+
+**FVM release**:
```
Comment (jennijuju) [inline content/en/developers/infrastructure/networks/index.md:176]: These information are wrong- please remove

File: content/en/developers/infrastructure/networks/index.md [hunk 8]
```diff
@@ -22,26 +21,184 @@ aliases:
...
+
+- [FVM M2.1 Sapphire (r08)](https://github.com/filecoin-project/ref-fvm/issues/940)
+- Lotus commit: [cb838cedae49106e9ceb7302add3688a835dbdb4](https://github.com/filecoin-project/lotus/commit/cb838cedae49106e9ceb7302add3688a835dbdb4)
+- [List of FVM releases](https://github.com/filecoin-project/ref-fvm/issues/692)
+
+**Resources**:
+
+- [Faucet](https://hyperspace.filtest.network/#faucet)
+- [Slack Channel for Updates: #fil-network-announcements](https://filecoinproject.slack.com/archives/C01AC6999KQ)
```
Comment (jennijuju) [inline content/en/developers/infrastructure/networks/index.md:185]:
```suggestion
- [Slack Channel for Updates/questions: #fil-net-hyperspace-discuss](https://filecoinproject.slack.com/archives/C01AC6999KQ)
```

File: content/en/developers/infrastructure/networks/index.md [hunk 9]
```diff
@@ -22,26 +21,184 @@ aliases:
...
+- [Slack Channel for Questions: #fil-help](https://filecoinproject.slack.com/archives/CEGN061C5)
```
Comment (jennijuju) [inline content/en/developers/infrastructure/networks/index.md:186]:
```

**Review by jennijuju — CHANGES_REQUESTED** [review]

**Pull request merged**